bundle/bamoe/manifests/bamoe-kogito-operator.clusterserviceversion.yaml

```diff
@@ -1003,5 +1003,5 @@ spec:
   maturity: beta
   provider:
     name: IBM
-  replaces: bamoe-kogito-operator.8.0.4-1
-  version: 8.0.4-2
+  replaces: bamoe-kogito-operator.8.0.4-3
```

Comment:
See comment on version.go file. I believe the replaces should be: `bamoe-kogito-operator.8.0.4-2`